From 9146999bce9a88e15b5f123d1aa1613926dd9a9c Mon Sep 17 00:00:00 2001
From: Colin Arnott <colin@urandom.co.uk>
Date: Thu, 8 Apr 2021 09:10:33 +0000
Subject: [PATCH] add module

It is not possible to build binaries without a module in recent versions
of the toolchain.
---
 go.mod | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 go.mod

diff --git a/go.mod b/go.mod
new file mode 100644
index 0000000..e045e05
--- /dev/null
+++ b/go.mod
@@ -0,0 +1,3 @@
+module github.com/jstemmer/gotags
+
+go 1.16
